### ISSUE TITLE
Tray mode not working with 0.8.X and Electron 8.1.1

### ISSUE THREAD

**Issue by vinifmor** [jiahaog/nativefier#925]
**Bug description**
I tried to install WhatsApp Web ( https://web.whatsapp.com ) and GMail ( https://mail.google.com ) using the **start-in-tray** parameter  ( **--tray=start-in-tray** ), but it seems not to be working anymore. No icon is attached to the system tray.

- [X] I looked at `nativefier --help` and https://github.com/jiahaog/nativefier/blob/master/docs/api.md , I'm sure I'm not misunderstanding expected behavior.
- [X] I searched existing open & closed issues. Yes, my bug is new.

**Steps to reproduce**
1. Run `nativefier https://web.whatsapp.com --tray=start-in-tray.`
2. Launch the application

**Expected behavior**
The application icon should be attached to the system tray

**Actual behavior**
No icon is attached, but the application keeps running on the background.

**Context**
 - Nativefier: 8.0.3
 - Node.js: 8.1.1
 - Npm: 6.13.4
 - OS: Manjaro Linux ( latest )
 - Kernel: 5.4.18

**Comment by ronjouch** [comment]
Thanks @vinifmor . By the way, are you affected by https://github.com/jiahaog/nativefier/issues/923 or https://github.com/jiahaog/nativefier/issues/924 ? Where & how do you install Nativefier? (Globally or locally? With/out sudo? What's your `$npm_config_prefix` ?)

**Comment by vinifmor** [comment]
No worries. I installed it locally ( no sudo ) in a separate Node environment and had no issues. It seems this env var **$npm_config_prefix** is not available in my end.

Installing it globally produces the same error discussed in #923.

```
> nativefier@8.0.3 postinstall /usr/lib/node_modules/nativefier
> cd app && echo '***** Installation will take 60s to finish, this is a bug ( https://github.com/jiahaog/nativefier/issues/923 ) that will be addressed in a future release. For now, have a cup of tea. *****' && npm install --no-package-lock --no-audit --silent

***** Installation will take 60s to finish, this is a bug ( https://github.com/jiahaog/nativefier/issues/923 ) that will be addressed in a future release. For now, have a cup of tea. *****
glob error [Error: EACCES: permission denied, scandir '/root/.npm/_logs'] {
  errno: -13,
  code: 'EACCES',
  syscall: 'scandir',
  path: '/root/.npm/_logs'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 243
npm ERR! nativefier@8.0.3 postinstall: `cd app && echo '***** Installation will take 60s to finish, this is a bug ( https://github.com/jiahaog/nativefier/issues/923 ) that will be addressed in a future release. For now, have a cup of tea. *****' && npm install --no-package-lock --no-audit --silent`
```

**Comment by ronjouch** [comment]
@vinifmor 8.0.3 install trouble is fixed in 8.0.4 🙂. And running `nativefier https://web.whatsapp.com --tray=start-in-tray` using 8.0.4, I get the expected behavior: app starts minimized, and I see the WhatsApp icon in my tray.

I see you use Manjaro, are you using Gnome Shell too? If so, are you familiar with Shell's tray shenanigans? You need to install libappindicator libs and extension [KStatusNotifierItem/AppIndicator Support](https://extensions.gnome.org/extension/615/appindicator-support/) for a functional tray.

Closing for now, feel free to reopen and bring additional debug info if I'm missing anything.

**Comment by vinifmor** [comment]
@ronjouch I was testing in XFCE. Unfortunately 8.0.4 produces the same behavior, but 7.7.1 works. For KDE, 8.0.4 works. Perhaps there is a new missing dependency that must be installed for this new Electron version. I will try to figure out here.

By the way, I develop an open source GUI for Linux for management of different types of applications / packages. The latest feature release ( 0.8.X ) is integrated with nativefier to generate Web Apps easily. If you want to have a look later -> https://github.com/vinifmor/bauh#native-web-applications--web-

I'm testing nativefier 8.X to raise the default version used by the tool.

Thanks for the great work.

Cheers
